fresque/gitview.py

```diff
@@ -219,3 +219,29 @@ def view_raw_file(repo, identifier, filename):
         headers['Content-Encoding'] = encoding
 
     return (data, 200, headers)
+
+
+@APP.route('/git/<repo>/tags/')
+@APP.route('/git/<repo>/tags')
+def view_tags(repo):
```

Comment:
@pypingou once the reviews for rpms gets closed and if contributor finds it ready for release for then he can tag it for release. Is it wrong?